### PR TITLE
Update ecs-deployment-tutorial.md

### PR DESCRIPTION
**Pull request by thisrohangupta**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by deepakbolangady — APPROVED** [review]

**Comment by thisrohangupta** [comment]
@deepakbolangady  Added some more content based on Pragyesh feedback.

**Review by deepakbolangady — DISMISSED** [review]
@thisrohangupta, I moved the **Update Green Service** content out of where it was initially added to where I think it can reside, alongside another option in that configuration window. I've also reworded it. Please take a look.
Cc @PragyeshMishra01, with whom I discussed these changes today.

Approving with these changes but keeping the PR open because this documentation requires Delegate 818xx to be released first.

**Review by deepakbolangady — DISMISSED** [review]

File: docs/continuous-delivery/deploy-srv-diff-platforms/aws/ecs/ecs-deployment-tutorial.md [hunk 1]
```diff
@@ -1261,6 +1277,34 @@ Lastly, the new ECS service is tagged with `BG_VERSION`, `BLUE`.
...
 
+#### ECS Blue Green service validation
+
+:::note
+
+Currently, ECS Blue Green service validation is behind the feature flag `CDS_ECS_BG_VALIDATION`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+
+:::
+
+ This feature flag adds a checkbox named **Update Green Service** that enables some validations to execute before the deployment. Before the deployment, Harness identifies ECS Blue and Green services based on the target group and updates tags accordingly. It then starts the deployment.
```
[the remark this answers is not in the document]
Comment (deepakbolangady) [reply docs/continuous-delivery/deploy-srv-diff-platforms/aws/ecs/ecs-deployment-tutorial.md]: Thanks, @PragyeshMishra01. I removed the text `some a checkbox **Update Green Service** that enables`.

**Review by deepakbolangady — APPROVED** [review]
Approving for merge

**Pull request merged**